packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -530,6 +562,45 @@ export function SelectPanel({
             >
               {footer}
             </Box>
+          ) : (
+            <Box
+              sx={
+                enabled
+                  ? undefined
+                  : {
+                      // .responsiveFooter
+                      display: ['flex', 'flex', 'none', 'none'],
+                      gap: 'var(--stack-gap-condensed)',
+                      justifyContent: 'right',
+                      padding: 3,
+                      // .Footer
+                      borderTop: 'var(--borderWidth-thin) solid',
+                      borderTopColor: 'var(--borderColor-default)',
+                    }
+              }
+              className={enabled ? clsx(classes.Footer, classes.ResponsiveFooter) : undefined}
```

Comment:
if this is net new code can we code it with css modules from the get-go? 🙏

Reply:
Oh we get to that now? I thought we can't yet because it's still behind a feature flag that we'd want the ability to roll back?

Reply:
Should be able to go all in with CSS for new features. The only precaution with having the flag fall back to styled components is that the new styles don't match the old ones. In this case there is nothing to compare the new styles to